File: MultiBoxClientJavaCore/src/sk/hackcraft/multibox/model/libraryitems/BackNavigationLibraryItem.java
package sk.hackcraft.multibox.model.libraryitems;

import sk.hackcraft.multibox.model.GenericLibraryItem;
import sk.hackcraft.multibox.model.Library;
import sk.hackcraft.multibox.model.LibraryItemType;

public class BackNavigationLibraryItem extends GenericLibraryItem
{
	public BackNavigationLibraryItem()
	{
		super(Library.BACK_NAVIGATION, LibraryItemType.BACK_NAVIGATION, "..");
	}
}

File: MultiboxClientCSharpCore/Model/LibraryItems/BackNavigationLibraryItem.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;


namespace MultiBoxCSharpCore
{
	class BackNavigationLibraryItem : GenericLibraryItem
	{
		public BackNavigationLibraryItem()
		: base(LibraryConstants.BackNavigation, LibraryItemType.BackNavigation, "..")
		{
		}
	}
}
